spec/turn_spec.rb

```diff
@@ -0,0 +1,40 @@
+require './lib/card'
+require './lib/turn'
+
+RSpec.describe Turn do
+    before :each do
+      @card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+      @turn = Turn.new("Juneau", @card)
+  end
+
+  it 'exists' do
+    expect(@turn).to be_instance_of(Turn)
+  end
+
+  it "returns a card" do
+    expect(@turn.card).to be_instance_of(Card)
+  end
+
+  it "returns an answer" do
+    expect(@turn.guess).to eq("Juneau")
+  end
+
+  it "returns if correct or not" do
+    @turn.card
+    @turn.guess
```

Comment:
These aren't really doing anything. You can take them out and the test will still work!